ansible/vars/all.yml

```diff
@@ -10,6 +10,8 @@ vagrant_local:
 php:
     install: '1'
     ppa: php
-    packages: [php7.0-cli, php7.0-intl, php7.0-mcrypt, php7.0-mysql, php7.0-curl, php7.0-json, php7.0-xml]
+    packages: [php7.0-cli, php7.0-intl, php7.0-mcrypt, php7.0-mysql, php7.0-curl, php7.0-json, php7.0-xml, php7.0-mbstring]
+    pecl_packages: [php-zip]
```

Comment:
@axelitus all done:
- don't install php-zip
- install git
- use composer flag `--prefer-source` to install packages using git
- increase composer timeout params, because with default value installation failed on my machine